frontend/packages/console-dynamic-plugin-sdk/@types/sdk/index.d.ts

```diff
@@ -1,6 +1,7 @@
 declare interface Window {
   SERVER_FLAGS: {
     basePath: string;
+    consoleVersion: string;
```

Comment:
@invincibleJai just wanted to strike up the same debate we have in the past: do we want to track the console version explicitly (since that is console specific? what does that mean for the HAC UI?). I see that this is used below to essentially help invalidate cached resources, so perhaps we simply rename/remap this to be a bit more generic?

Reply:
yeah nice point, currently `window. SERVER_FLAGS. consoleVersion` is being set by OCP backend [0] and the explicit cache invalidation logic is based on it and sure it won't make sense to have it in SDK as HAC no way will be having it. So it does not make sense to have it here. 

https://github.com/openshift/console/blob/master/pkg/server/server.go#L656

Have removed it and not perform explicit cache validation based on consoleVersion and it does that auto for now i.e it does update the redux with resources in LS, remove the  LS key and make the call to openApi again and thus new response would be cached

Reply:
@invincibleJai do we know without the flag how many times we'll make the request to update the resource? I think doing it auto should be ok for now, but I'm assuming we added the check for the console version to avoid making too many calls.

Reply:
the number of calls are unchanged, as we not invalidating cache so will return with cached data and in both cases, `startAPIDiscovery` is being called

https://github.com/openshift/console/pull/10903/files#diff-0403e41025b34d5ee8caf5b6c27bc0b642a8c52e567111e015a5e28430a5304dR171-R177

The console version does not come from openApi call but the OCP backend sets it and SDK should be agnostic of it as HAC backend may not provide that.